Sources/NIOKit/NIOKitTestCase.swift

```diff
@@ -0,0 +1,35 @@
+import NIO
+import XCTest
+
+protocol NIOKitTestCaseProtocol: class {
```

Comment:
Do we need this protocol or can we do everything with the sub-class?

Reply:
I'll check :)

Reply:
See new PR